### PR TITLE
feat(Dash Radio): change dash radio to litt live

### LINKED ISSUE #8105: Dash/LITT | littlive.com
### Presence name

dash

### Description

Dash radio has been rebranded to LITT radio, so the presence is outdated and doesn't work anymore. I would like to see it updated to use the newer branding and links.

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bakzkndd**
## Description 

Updated metadata to support matches for v3. Also updated the Dash Radio presence to LITT Live, because Dash Radio doesn't exist anymore and is now LITT Live. Also added a feature to it which gets the current radio logo if it can, otherwise it just uses the logo. This way it is clearer for other users which radio is being listened to without causing clutter.

* Resolves #8105 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format` (didn't do this since it updated 1k+ metadata files, checked in the discord to see what i had to do and was told to just leave it be for now)
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image

</details>

### CODE REVIEW

**Comment by bakzkndd** [comment]
I've run `yarn lint` instead of `yarn format`

**Review by theusaf — CHANGES_REQUESTED** [review]
Revert changes to TomsHardware and Tweakers, since they aren't relevant to this pr

File: websites/L/LITT Live/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		"id": "202236922217758721"
+	},
+	"service": "LITT Live",
+	"description": {
+		"en": "100% Free Music. More Variety. Zero Commercials. The Best DJ's & Stations. No BS",
+		"nl": "100% gratis muziek. Meer afwisseling. Geen reclames. De beste dj's en stations. Geen BS"
+	},
+	"url": "littlive.com",
+	"version": "1.1.0",
+	"logo": "https://cdn.discordapp.com/app-assets/575756169986048004/1211339369386999808.png",
```
Comment (theusaf) [inline websites/L/LITT Live/metadata.json:14]:
```suggestion
	"logo": "https://cdn.discordapp.com/app-assets/575756169986048004/1211339369386999808.png?size=512",
```

File: websites/L/LITT Live/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	"thumbnail": "https://cdn.discordapp.com/app-assets/575756169986048004/1211339525565976606.png",
```
Comment (theusaf) [inline websites/L/LITT Live/metadata.json:15]: > The [thumbnail](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#thumbnail) should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.

File: websites/L/LITT Live/presence.ts [hunk 3]
```diff
@@ -9,12 +9,16 @@ const presence = new Presence({
 let songName: HTMLElement,
 	songArtist: HTMLElement,
 	songNameS: string,
-	songArtistS: string;
+	songArtistS: string,
+	thumbnail: HTMLImageElement;
 
 presence.on("UpdateData", async () => {
+	if (document.querySelectorAll(".MuiPaper-elevation")[0] && document.querySelectorAll(".MuiPaper-elevation")[0].childNodes[1]) 
+		thumbnail = document.querySelectorAll(".MuiPaper-elevation")[0].childNodes[1].childNodes[0].childNodes[0].childNodes[0] as HTMLImageElement;
```
Comment (theusaf) [inline websites/L/LITT Live/presence.ts:17]: * `querySelectorAll(...)[0]` can be simplified to `querySelector(...)`.
* Considering the multiple uses of the same selector, assign the element to a variable.

File: websites/L/LITT Live/presence.ts [hunk 4]
```diff
@@ -9,12 +9,16 @@ const presence = new Presence({
...
+	
 	const presenceData: PresenceData = {
 		largeImageKey:
-			"https://cdn.rcd.gg/PreMiD/websites/D/Dash%20Radio/assets/logo.png",
+			thumbnail ?? "logo",
```
Comment (theusaf) [inline websites/L/LITT Live/presence.ts:21]:
```suggestion
			thumbnail ?? "https://cdn.discordapp.com/app-assets/575756169986048004/1211339369386999808.png?size=512",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by bakzkndd** [comment]
> Make sure to resolve merge conflicts

I can't see the conflict, as in I can see thaat it says that the dash radio/metadata is a conflicting file, but I can't see why, or do anything about it seemingly.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**